lib/helperFunctions.js

```diff
@@ -1,7 +1,91 @@
+import axios from "axios";
+
 export const scrollToSection = (id) => {
   const element = document.getElementById(id);
   const yOffset = -70; // Adjust this value based on your header height
   const y = element.getBoundingClientRect().top + window.scrollY + yOffset;
 
   window.scrollTo({ top: y, behavior: "smooth" });
 };
+
+const parseOriginFromUrl = (url) => {
+  /**
+   * splits https://api.github.com/repos/org-name/repo-name/issues/25
+   * into ["https://", "api.github.com", "repos", "org-name", "repo-name", "issues", "25"]
+   */
+  const parts = url.split(/https:\/\/|\//gm)
+  const organization = parts[3]
+  const repo = parts[4]
+  /**
+   * accessing a github profile or organization and adding a .png 
+   * at the end of the URL will return their logo/profile picture
+   */
+  const logoUrl = `https://github.com/${organization}.png`
+
+  return {
+    organization,
+    repo,
+    logoUrl
+  }
+}
+
+export async function fetchContributions () {
+  try {
+    const response = await axios.get(
+      "https://api.github.com/search/issues",
+      {
+        params: {
+          q: "author:A-4-Atom",
+          sort: "created",
+          order: "desc",
+          //Set How many latest items to fetch issues+pull_requests combined
+          per_page: 12, 
```

Comment:
This also a variable

Reply:
Made a new variable in constants/index file named itemsPerPage